### ISSUE TITLE
make_travis_yml_2.hs scripts broken under Windows

### ISSUE THREAD

**Issue by bezirg** [hvr/multi-ghc-travis#103]
I use Windows to generate the scripts. Because of using `System.FilePath.</>` the generated .travis.yml are using windows backslashes instead of unix forward slashes.

Example Travis error of Windows-generated make-travis script:

```
The command "(cd "."; cabal sdist)" exited with 0.
0.01s
$ mv "."\dist\hlogo-*.tar.gz ${DISTDIR}/
mv: cannot stat ‘.disthlogo-*.tar.gz’: No such file or directory


The command "mv "."\dist\hlogo-*.tar.gz ${DISTDIR}/" exited with 1.
```

**Comment by bezirg** [comment]
Perhaps the solution would be:

In `make_travis_yml_2.hs` substitute:

```
import System.FilePath ((</>), takeDirectory, takeFileName)
```

with 

```
import System.FilePath.Posix ((</>), takeDirectory, takeFileName)
```

**Comment by phadej** [comment]
Please make a PR

**Comment by phadej** [comment]
Fixed in #104 + #107
